x-pack/legacy/plugins/apm/public/components/shared/charts/CustomPlot/AnnotationsPlot.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { VerticalGridLines } from 'react-vis';
+import theme from '@elastic/eui/dist/eui_theme_light.json';
+import {
+  EuiIcon,
+  EuiToolTip,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { Maybe } from '../../../../../typings/common';
+import { Annotation } from '../../../../../common/annotations';
+import { PlotValues, SharedPlot } from './plotUtils';
+import { asAbsoluteDateTime } from '../../../../utils/formatters';
+
+interface Props {
+  annotations: Annotation[];
+  plotValues: PlotValues;
+  width: number;
+  overlay: Maybe<HTMLElement>;
+}
+
+const style = {
+  stroke: theme.euiColorSecondary,
+  strokeDasharray: 'none'
+};
+
+export function AnnotationsPlot(props: Props) {
+  const { plotValues, annotations } = props;
+
+  const tickValues = annotations.map(annotation => annotation.time);
+
+  return (
+    <>
+      <SharedPlot plotValues={plotValues}>
+        <VerticalGridLines tickValues={tickValues} style={style} />
+      </SharedPlot>
+      {annotations.map(annotation => (
+        <div
+          key={annotation.id}
+          style={{
+            position: 'absolute',
+            left: plotValues.x(annotation.time) - 8,
+            top: -2
+          }}
+        >
+          <EuiToolTip
+            title={asAbsoluteDateTime(annotation.time, 'seconds')}
+            content={
+              <EuiFlexGroup>
+                <EuiFlexItem grow={true}>
+                  <EuiText color="subdued">
```

Comment:
I think it's better to drop the `subdued` since it renders quite dark in these tooltips.

<img width="336" alt="Screenshot 2019-12-11 at 09 07 59" src="https://user-images.githubusercontent.com/4104278/70603157-35fe8980-1bf6-11ea-832a-31981e5deab5.png">

Reply:
Here's how it looks now, WDYT?

![image](https://user-images.githubusercontent.com/352732/70610779-7402aa00-1c04-11ea-8812-60586b455f11.png)

Reply:
LGTM 👍